File: SeengWook/CPP/MATH_2/1978.cpp
#include <iostream>

using namespace std;

int main(){
	
	int n;
	int cnt = 0;
	cin >> n;
	
	for(int i=0; i<n; i++){
		int number;
		bool cmp = true;
		cin >> number;
		
		if(number != 1){
			cout << number << endl;
			for(int j=1; j<number; j++){
				if(number%j == 0 && j != 1){
					cmp = false;
					break;
				}
			}
		}
		if(cmp && number != 1) cnt++;
	}
	cout << cnt;
	return 0;
}

File: JuHo/1_py/math2_py/1978.py
n = int(input())
number = map(int, input().split())
sosu = 0
for num in number:
    cnt = 0
    if num > 1:
        for i in range(2, num):
            if num % i == 0:
                cnt += 1
        if cnt == 0:
            sosu += 1
print(sosu)
